.drone.yml

```diff
@@ -35,14 +35,23 @@ pipeline:
   #     branch: master
 
   publish:
-    image: plugins/docker
-    repo: drone/drone
-    username: ${DOCKER_USERNAME}
-    password: ${DOCKER_PASSWORD}
-    tag: [ 0.6 ]
-    when:
-      branch: master
-      event: push
+
+    docker:
```

Comment:
Ooh, I didn't know you could nest other steps under each step in the pipeline, that is neat.